### PR TITLE
APP-3106-1 JIRA: Allow custom fields to be copied to issues from requirements

### PR DESCRIPTION
**Pull request by jemmyw**

### CODE REVIEW

File: lib/services/jira.rb [hunk 1]
```diff
@@ -378,12 +383,16 @@ def update_issue(issue_info, resource, initiative, version, parent)
 
     if @feature == resource
       # Only update custom_fields and due dates for features, not requirements.
-      # This will cause an issue with two-way field syncing if we constantly overwrite requirements' custom fields
+      # This will cause an issue with two-way field syncing if we constantly
+      # overwrite requirements' custom fields
       issue.fields
         .merge!(mapped_custom_fields(@feature, issue_type))
```
Comment (JeremyGeros) [inline lib/services/jira.rb]: This line should be removed now since the lines you added below do exactly the same thing

Comment (jemmyw) [inline lib/services/jira.rb]: It doesn't do exactly the same though, but I'm not sure of the difference or why this was like this in the first place.. If resource is a requirement this line runs the custom field mapping against the feature.

Comment (JeremyGeros) [inline lib/services/jira.rb]: That is only for create, for update this is only run when the resource == @feature, not the same as the create code where the custom fields from the feature are added to a requirement.

Comment (jemmyw) [inline lib/services/jira.rb]: OK, removed

**Pull request merged**